src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
```

Comment:
Ensure that the padding value is consistent with the design requirements. If the design specifies different padding, consider using variables for consistency and ease of maintenance.